packages/dds/tree/src/test/shared-tree/fuzzEditGenerator.ts

```diff
@@ -65,6 +66,11 @@ export interface TreeContext {
 	treeIndex: number;
 }
 
+export interface PathWithNodeRange {
+	path: UpPath;
+	nodesToDelete: number;
```

Comment:
I think `count` would be a better name for this field.
This interface doesn't have to know what will happen to the nodes (for example, we'll likely want to reuse this interface when adding fuzzing for move operations).
Also, the "nodes" part of `nodesToDelete` would be more appropriate if `nodesToDelete` contained the data for the nodes or a path to it. It's obvious from the number type that this is not the case, but it's nice to be precise.

Reply:
renamed `nodesToDelete` to `count